docs/resources/add-ins-differences.md

```diff
@@ -1,41 +1,39 @@
+:::image type="content" source="../images/office-programmability-diagram.png" alt-text="A four-quadrant diagram showing the focus areas for different Office extensibility solutions. Both Office Scripts and Office Web Add-ins are focused on cross-platform experiences and collaboration. Office Scripts cater to end users, whereas Office Web Add-ins target professional developers.":::
 
 Office Scripts run to completion with a manual button press or as a step in [Power Automate](https://flow.microsoft.com/), whereas Office Add-ins continue running depending on how they are configured. For example, you can configure an Office Add-in to continue running even when its task pane is closed. This means that Office Add-ins maintain state during a session, whereas Office Scripts don't maintain an internal state between runs. If the solution you are building requires a maintained state, you should visit the [Office Add-ins documentation](/office/dev/add-ins) to learn more about Office Add-ins.
 
 The rest of this article describes on the main differences between Office Add-ins and Office Scripts.
 
 ## Platform Support
 
-Office Add-ins are cross-platform. They work across Windows desktop, Mac, iOS, and web platforms and provide the same experience on each. Any exception to this is noted in the documentation of the individual API.
+The following shows which features are supported by which platforms and products.
 
-Office Scripts are currently only supported by for Excel on the web. All recording, editing, and script management is done on the web platform.
-
-### Script support for Excel on Windows
-
-[!INCLUDE [Run-from-button support](../includes/run-from-button-desktop-support.md)]
+[!INCLUDE [Platform support table](../includes/platform-support-table.md)]
 
 ## APIs
 
 While the Office JavaScript APIs for Office Add-ins and the Office Scripts APIs share some functionality, they are different platforms. The Office Scripts APIs are an optimized, synchronous subset of the Excel JavaScript API model. The major difference is usage of the `load`/`sync` paradigm with add-ins. Additionally, add-ins offer APIs for events and a broader set of functionality outside of Excel, known as the Common APIs.
 
+Office Add-ins offer greater connectivity to external web services and libraries. Office Scripts are limited to [specific external calls](../develop/external-calls.md).
+
 ### Events
 
 Office Scripts do not support workbook-level [events](/office/dev/add-ins/excel/excel-add-ins-events). Scripts are either triggered by users selecting the **Run** button for a script or through Power Automate. Every script runs the code in a single `main` function, then ends.
 
-### Common APIs
+### UX/UI controls
```

Comment:
General screenreader and accessibility guidance is to type out "and" or "or" instead of using a slash, unless referring to a broadly accepted terms such as "CD/DVD drive" or "on/off button". 

I think UX/UI as a term is in a hazy middle ground for our audience – it's somewhat common. Personally I think "UX and UI" is more accessible but I didn't find any explicit guidance for this term. Sharing this all for your consideration.

If you change this instance then I also suggest changing the term in the following paragraph.
```suggestion
### UX and UI controls
```